web_widget_open_tab/static/src/js/widget.js

```diff
@@ -0,0 +1,82 @@
+odoo.define('web_widget_open_tab.FieldOpenTab', function(require) {
+    "use strict";
+
+    var AbstractField = require('web.AbstractField');
+    var field_registry = require('web.field_registry');
+    var ListRenderer = require('web.ListRenderer');
+    var core = require('web.core');
+    var config = require('web.config');
+    var qweb = core.qweb;
+    var _t = core._t;
+
+    var FieldOpenTab = AbstractField.extend({
+        description: "",
+        supportedFieldTypes: ['integer'],
+        events: _.extend({}, AbstractField.prototype.events, {
+            'click': '_onClick',
+        }),
+        isSet: function () {
+            return true;
+        },
+        _getReference: function () {
+            var url = new URL(window.location.href);
```

Comment:
what about using https://developer.mozilla.org/en-US/docs/Web/API/URLSearchParams ? 
Not a blocker, just check if you can do it easier w/ it :wink:

Reply:
I cannot use URLSearchParams as the original url of odoo is `/web#...` This is not an standard url that URLSearchParams can process :thinking:

Reply:
well, you should take only the part after the hash. In any case not a blocker...

Reply:
quick example:

![image](https://user-images.githubusercontent.com/347149/87136024-2eda4f80-c29b-11ea-97a3-cea41784aa61.png)

Reply:
Thanks, I didn't know this, much easier now :wink: